README.rst

```diff
@@ -1,39 +1,54 @@
+Dependencies
+=============
 
-Introduction
-============
+This driver depends on the `Register <https://github.com/adafruit/Adafruit_CircuitPython_Register>`_
+and `Bus Device <https://github.com/adafruit/Adafruit_CircuitPython_BusDevice>`_
+libraries. Please ensure they are also available on the CircuitPython filesystem.
+This is easily achieved by downloading
+`a library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
 
-.. image:: https://readthedocs.org/projects/adafruit-circuitpython-bno055/badge/?version=latest
-    :target: https://circuitpython.readthedocs.io/projects/bno055/en/latest/
-    :alt: Documentation Status
+Usage Notes
+===========
 
-.. image :: https://badges.gitter.im/adafruit/circuitpython.svg
-    :target: https://gitter.im/adafruit/circuitpython?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
-    :alt: Gitter
+Of course, you must import the library to use it:
 
-TODO
+.. code:: python
 
-Dependencies
-=============
-This driver depends on:
+    import bno055
 
-* `Adafruit CircuitPython <https://github.com/adafruit/circuitpython>`_
-* `Bus Device <https://github.com/adafruit/Adafruit_CircuitPython_BusDevice>`_
 
-Please ensure all dependencies are available on the CircuitPython filesystem.
-This is easily achieved by downloading
-`the Adafruit library and driver bundle <https://github.com/adafruit/Adafruit_CircuitPython_Bundle>`_.
+All the Adafruit RTC libraries take an instantiated and active I2C object (from
+the `nativeio` or the `bitbangio` library) as an argument to their constructor.
+The way to create an I2C object depends on the board you are using. For boards
+with labeled SCL and SDA pins, you can:
 
-Usage Example
-=============
+.. code:: python
+
+    from bitbangio import I2C
+    from board import SDA, SCL
+
+    i2c = I2C(SCL, SDA, frequency=1000)
+
+
+Note that because we can't control the clock stretching timeout in
+CircuitPython, you have to set the I2C clock speed to a fairly low value for
+reliable communication.
+
+Once you have the I2C object, you can create the sensor object:
+
+.. code:: python
+
+    sensor = bno055.BNO055(i2c)
+
+
+And then you can start reading the measurements:
 
-TODO
+.. code:: python
 
-Contributing
-============
+    print(sensor.temperature)
+    print(sensor.euler)
+    print(sensor.gravity)
 
-Contributions are welcome! Please read our `Code of Conduct
-<https://github.com/adafruit/Adafruit_CircuitPython_bno055/blob/master/CODE_OF_CONDUCT.md>`_
-before contributing to help this project stay welcoming.
```

Comment:
K no worries. As a heads up, I'm traveling to Europe on Wednesday and will be online sporadically for the next week or so after that.